core/app/models/spree/variant.rb

```diff
@@ -87,6 +87,23 @@ def self.active(currency = nil)
       joins(:prices).where(deleted_at: nil).where('spree_prices.currency' => currency || Spree::Config[:currency]).where('spree_prices.amount IS NOT NULL')
     end
 
+    # Returns default pricing options that can easily be used in scopes and instance methods
+    # This method is need in scopes and on instances.
+    #
+    # @return [Spree::Variant::Pricer::PricingOptions] the default pricing options
+    def self.default_pricing_options
+      @default_pricing_options ||= Spree::Config.pricing_options_class.new
```

Comment:
Would this make more sense on `Spree::Config`. Having class instance variables behaves strangely in development mode as classes are regularly reloaded.